dashboard/app.py

```diff
@@ -124,7 +117,7 @@
 def check_access():
     """Users can only view their own security alerts."""
     user = User(session)
-    if request.args.get("channel") == Userhash():
+    if request.args.get("channel") == user.hash():
```

Comment:
Quick comment to attract your attention that I am not adding the wrong thing here.